confiture-web-app/src/components/report/ReportCriteria.vue

```diff
@@ -0,0 +1,141 @@
+<script setup lang="ts">
+import { computed } from "vue";
+
+import { pluralize } from "../../utils";
+
+// FIXME: type things
+const props = defineProps<{
+  pagesData: any;
+  transverseData: any;
+  showFilters?: boolean;
+}>();
+
+const dataCount = computed(() => {
+  return (
+    props.pagesData
+      .map((page: any) => page.topics.map((topic: any) => topic.improvements))
+      .flat(2).length + props.transverseData.length
+  );
+});
+</script>
+
+<template>
+  <div class="main">
+    <div class="sidebar">
+      <nav class="fr-sidemenu fr-mb-3w" aria-label="Liste des pages">
+        <div class="fr-sidemenu__inner">
+          <button
+            class="fr-sidemenu__btn"
+            hidden
+            aria-controls="fr-sidemenu-wrapper"
+            aria-expanded="false"
+          >
+            Pages
+          </button>
+          <div id="fr-sidemenu-wrapper" class="fr-collapse">
+            <div class="fr-sidemenu__title fr-mb-2w">Pages</div>
+            <ul class="fr-sidemenu__list">
+              <li
+                v-if="transverseData.length"
+                :class="[
+                  'fr-sidemenu__item',
+                  {
+                    'fr-sidemenu__item--active': Boolean(transverseData.length)
```

Comment:
C'est plus court et plus lisible de direct comparer à 0 IMO
```suggestion
                    'fr-sidemenu__item--active': transverseData.length > 0
```